eng/ApiCompat/ApiCompat.csproj

```diff
@@ -0,0 +1,47 @@
+﻿<Project Sdk="Microsoft.NET.Sdk" DefaultTargets="VerifyCompat">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <RunApiCompatForSrc>true</RunApiCompatForSrc>
+    <IntermediateOutputPath>$(IntermediateOutputPath)/$(TargetPackageName)</IntermediateOutputPath>    
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.DotNet.ApiCompat" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Azure.Core" />
+    <PackageReference Include="Azure.Data.AppConfiguration" />
+    <PackageReference Include="Azure.Identity" />
+    <PackageReference Include="Azure.Security.KeyVault.Certificates" />
+    <PackageReference Include="Azure.Security.KeyVault.Keys" />
+    <PackageReference Include="Azure.Security.KeyVault.Secrets" />
+    <PackageReference Include="Azure.Storage.Blobs" />
+    <PackageReference Include="Azure.Storage.Blobs.Batch" />
+    <PackageReference Include="Azure.Storage.Common" />
+    <PackageReference Include="Azure.Storage.Files.Shares" />
+    <PackageReference Include="Azure.Storage.Queues" />
+    <PackageReference Include="Microsoft.Extensions.Azure" />
+  </ItemGroup>
+
+  <Import Project="..\Packages.Data.props" />
+
+  <Target Name="VerifyCompat" DependsOnTargets="_ResolveResolvedMatchingContract;$(TargetsTriggeredByCompilation)">
+  </Target>
+
+  <Target Name="_ResolveResolvedMatchingContract" DependsOnTargets="ResolveReferences">
+    <PropertyGroup>
+      <TargetPackageDll Condition="'%(ResolvedCompileFileDefinitions.NuGetPackageId)' == '$(TargetPackageName)'" >%(ResolvedCompileFileDefinitions.Identity)</TargetPackageDll>
+    </PropertyGroup>
+
+    <ItemGroup>
+      <ResolvedMatchingContract Include="$(TargetPackageDll)" />
+      <_DependencyDirectories Include="$(TargetOutputPath)" />
+      <ReferencePath Remove="$(TargetPackageDll)" />
+    </ItemGroup>
+    <Error Condition="'@(ResolvedMatchingContract)' == ''"
```

Comment:
Would a person that doesn't know about what ApiCompat is understand this message?

Reply:
Added more detail